### ISSUE TITLE
HTTP 500 when loading contacts details page after upgrade to 5.0.1

### ISSUE THREAD

**Issue by minchaminder** [mautic/mautic#13197]
### Mautic Version

5.0.x series

### PHP version

8.1.27

### What happened?

A bug happened! I'm getting a 500 error only when loading the contacts details page like `/s/contacts/view/18610`

```
[2024-01-10 18:14:19] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Key "address2" for array with keys " ///// does not exist. at /var/www/html/app/bundles/LeadBundle/Resources/views/Lead/lead.html.twig:431)"} {"hostname":"mautic","pid":36779}
```
**Solved by toggling address2 in /s/contacts/fields to on** 

### How can we reproduce this issue?

Step 1: Upgrade 

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by escopecz** [comment]
Did you remove the Address 2 custom field?

**Comment by dsp76** [comment]
Looks like the original poster had it disabled before and fixed it with enabling it again. 
Those fields can be disabled:

![image](https://github.com/mautic/mautic/assets/1241376/24518168-689c-46c9-a26d-0d1aa875cb5b)

Is it not optional anymore in Mautic 5?

**Comment by escopecz** [comment]
It is still a bug but I'm glad we found a workaround until it's fixed.

**Comment by minchaminder** [comment]
> Did you remove the Address 2 custom field?

I did not do any changes, so likely that it was toggled off before the upgrade but did not cause any issues

**Comment by mollux** [comment]
@minchaminder @dsp76 @escopecz 

it's indeed a bug/regression caused by the twig migration. I addressed this in #13254

**Comment by escopecz** [comment]
The fix was confirmed. Will be released in 5.0.3. Thank you Mattias!
